hw2/03-count.js

```diff
@@ -1,3 +1,15 @@
-// Add your code here
-
-input.addEventListener('keydown', handleKeyDown);
+document.addEventListener('DOMContentLoaded', function () {
+    const input = document.getElementById('input-field') //Get the input field
+    input.addEventListener('input', function (event) {
+        //Changed keydown to input - wasn't working consistently with keydown
+        let targetString = event.target.value //Get input string
+        if (targetString.length > 0) {
+            //Verify string length - otherwise it highlights spaces
+            let textBody = document.getElementById('text-body') //Get the text
+            //Regex: \b = word boundary, g = global, i = case insensitive
+            let regExp = new RegExp(`\\b${targetString}\\b`, 'gi') //Couldn't get this to work with a literal - suspect the variable breaks it
+            let newText = textBody.textContent.replace(regExp, `<mark>$&</mark>`) //Add <mark> tags
```

Comment:
Using the mark tag changes spacing I found. You can use a span style to change it inline instead and it won't move the text around.

Reply:
Here's the syntax. You'll just put your styling inside the quotes and pop this span in where you have the mark. 
<span style="">$&</span>

https://www.w3schools.com/tags/tag_span.asp#:~:text=The%20tag%20is%20an,span%3E%20is%20an%20inline%20element.

Reply:
You can use span and set bg-warning class as the background color which is the style from bootstrap